src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+import './TodoList.scss';
```

Comment:
Consider organizing imports alphabetically for better readability and maintainability. 📚